src/main/target/common_pre.h

```diff
@@ -28,7 +28,7 @@
 // -Wpadded can be turned on to check padding of structs
 //#pragma GCC diagnostic warning "-Wpadded"
 
-//#define SCHEDULER_DEBUG // define this to use scheduler debug[] values. Undefined by default for performance reasons
+#define SCHEDULER_DEBUG // define this to use scheduler debug[] values. Undefined by default for performance reasons
```

Comment:
Should this be enabled all of the time? One potential avenue is to make this define conditional to `DEBUG` to keep this being built for at least one target in CI.

Also, how does this relate / interact with `USE_LATE_TASK_STATISTICS`?

Reply:
Hangover from prior code. Removed. It was simply removing the conditional checks for DEBUG_SET macros in the scheduler loop, but the impact of those if negligible. `USE_LATE_TASK_STATISTICS` adds extra stats into the `task_t` structure which we exclude for smaller target FLASH.